whatToDad_site/whatToDad_app/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.1.4 on 2022-12-27 23:55
```

Comment:
If this was a live site, then deleting and recreating already checked in migrations that were deployed would be a bad idea. In production you would rely on these being stable so it could update safely from the previous state.